src/main/scala/firrtl/passes/wiring/WiringUtils.scala

```diff
@@ -87,9 +100,9 @@ object WiringUtils {
   /** Counts the number of instances of a module declared under a top module
     */
   def countInstances(childrenMap: ChildrenMap, top: String, module: String): Int = {
-    if(top == module) 1
-    else childrenMap(top).foldLeft(0) { case (count, (i, child)) =>
-      count + countInstances(childrenMap, child, module)
+    if (top == module) { 1 }
+    else { childrenMap(top).foldLeft(0) { case (count, (i, child)) =>
+      count + countInstances(childrenMap, child, module) }
     }
```

Comment:
What is this curly brace closing? Seems weird that it's lined up with the else but isn't closing the else?

Reply:
Should be the added `{` for the `else`. Adding all these isn't doing anything... However, I realize that this method isn't actually used anymore. I'm trying to aggressively remove deprecated portions of WiringUtils that are relying on Lineage/ChildrenMap. I'll rip this whole thing out. That _may_ have implications for others that are depending on portions of WiringUtils existing...